### ISSUE TITLE
Release schedule?

### ISSUE THREAD

**Issue by sthartman** [romerogroup/pyprocar#80]
Hello,
I noticed that PyProcar hasn't had a release since March 2021, resulting in the pip/conda versions being quite out of date compared to the online documentation. Do you think you could put out a new release soon?

Thanks

**Comment by ahromero** [comment]
Yes Steven, we will have  a major release very soon, we are finishing the documentation… we have improved the package quite a lot and it has now more properties and it is faster..


______________________________________::::__________________________________________
Prof. Aldo Humberto Romero
Director Research Computing, West Virginia University
Eberly Family Distinguished Professor
Fellow American Physical Society
Assistant Editor EPJB
Section Editor Papers in Physics
Member of the Editor Board of  “Materials”

Physics and Astronomy Department
West Virginia University
135 Willey Street, PO Box 6315
111 White Hall
Morgantown, WV 26506
Phone:  304-293-6317
Fax:  304-293-5732
email: ***@***.***


From: Steven Hartman ***@***.***>
Date: Wednesday, August 10, 2022 at 5:04 PM
To: romerogroup/pyprocar ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [romerogroup/pyprocar] Release schedule? (Issue #80)

Hello,
I noticed that PyProcar hasn't had a release since March 2021, resulting in the pip/conda versions being quite out of date compared to the online documentation. Do you think you could put out a new release soon?

Thanks
